app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const { getCategories } = require("./Controllers/getCategories.controller");
+const app = express();
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
+app.all("/*", (req, res) => {
```

Comment:
you've added a 404 handler but haven't written a test for it, please make sure you are testing this.